package.json

```diff
@@ -1,7 +1,9 @@
 {
   "private": true,
   "version": "3.0.0-alpha.14.5",
-  "dependencies": {},
+  "dependencies": {
+    "cypress": "^3.1.2"
```

Comment:
It should be a `devDependencies`

Reply:
https://github.com/strapi/strapi/blob/d60b5c5b34d4aa72c43083460ad89c927485e487/package.json#L9-L11